integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/database/device/DlmsDeviceSteps.java

```diff
@@ -223,7 +228,7 @@ public void theDlmsDeviceWithIdentificationDoesNotExist(final String deviceIdent
         assertThat(device).as("DLMS device with identification " + deviceIdentification + " in core database").isNull();
     }
 
-    @Then("^the new keys are stored in the osgp_adapter_protocol_dlms database security_key table$")
+    @Then("^the new keys are stored in the secret management database encrypted_secret table$")
     public void theNewKeysAreStoredInTheOsgpAdapterProtocolDlmsDatabaseSecurityKeyTable() {
```

Comment:
Rename the method in same manner

Reply:
My bad. Changed it in another place, but not here. Done now.